modules/apps/jaggery/mdm/src/mdm/units/notification-listing/notification-listing.hbs

```diff
@@ -15,13 +15,13 @@
                 <div class="wr-advance-operations">
                     <div class="row no-gutter">
                         <div class="wr-hidden-operations-nav col-lg-4">
-                            <a href="javascript:void(0)" onclick="showAdvanceOperation('unread', this)" class="selected">
+                            <a id=unReadNotifications href="javascript:void(0)" onclick="showAdvanceOperation('unread', this)" class="selected">
```

Comment:
unquoted attributes are valid on html5 specification, but as a best practice do recommend to use quotes.